lib/components/camera.dart

```diff
@@ -36,7 +39,23 @@ class Camera {
   }
 
   Future<PickedFile> _imageSelectorCamera() async {
-    var imageFile = await picker.getImage(source: ImageSource.camera);
-    return imageFile;
+    return picker.getImage(source: ImageSource.camera);
+  }
+
+  Future<void> _deleteIncorretFile() async {
+    final directory = new Directory(ANDROID_DIRECTORY_PATH);
+
+    var allFiles = directory.list();
+
+    var incorrectFiles = await allFiles.where((file) {
+      return !file.path
+              .substring(ANDROID_DIRECTORY_PATH.length + 1)
+              .startsWith('20') &&
+          file.path.contains(new RegExp(r'\.(png|jpe?g)'));
```

Comment:
n precisa mais colocar a palavra new em Dart, se vc viu isso em tutorial é pq o vídeo é antigo
```suggestion
          file.path.contains(RegExp(r'\.(png|jpe?g)'));
```